judge/comments.py

```diff
@@ -98,6 +105,7 @@ def get_context_data(self, **kwargs):
             unique_together_left_join(queryset, CommentVote, 'comment', 'voter', profile.id)
             context['is_new_user'] = (not self.request.user.is_staff and
                                       not profile.submission_set.filter(points=F('problem__points')).exists())
+            context['comment_lock'] = self.is_comment_locked()
```

Comment:
As I understand it this would cause the UI to show comments as unlocked for users that are not logged in. It wouldn't be an issue since they wouldn't be able to post comments regardless, but I think it would be better to be consistent and tell them that comments are locked anyway.